src/olympia/search/filters.py

```diff
@@ -327,29 +345,44 @@ def filter_queryset(self, request, qs, view):
 class SearchParameterFilter(BaseFilterBackend):
     """
     A django-rest-framework filter backend for ES queries that look for items
-    matching a specific set of fields in request.GET: app, appversion,
-    platform, tag and type.
+    matching a specific set of params in request.GET: app, appversion,
+    author, category, exclude_addons, platform, tag and type.
     """
     available_filters = [AddonAppFilterParam, AddonAppVersionFilterParam,
-                         AddonPlatformFilterParam, AddonTypeFilterParam,
-                         AddonCategoryFilterParam, AddonTagFilterParam,
-                         AddonAuthorFilterParam]
+                         AddonAuthorFilterParam, AddonCategoryFilterParam,
+                         AddonPlatformFilterParam, AddonTagFilterParam,
+                         AddonTypeFilterParam]
 
-    def filter_queryset(self, request, qs, view):
-        must = []
+    available_excludes = [AddonExcludeAddonsExcludeParam]
 
-        for filter_class in self.available_filters:
+    def get_applicable_clauses(self, request, params_to_try):
+        rval = []
```

Comment:
I'm a fan of naming this `return_value` or at least `retval` if it doesn't fit easily into 79 limit.